python/ray/worker.py

```diff
@@ -602,6 +602,7 @@ def init(
         include_dashboard: Optional[bool] = None,
         dashboard_host: str = ray_constants.DEFAULT_DASHBOARD_IP,
         dashboard_port: Optional[int] = None,
+        gcs_server_port: Optional[int] = None,
```

Comment:
Can this be prefixed with `_`? I wonder why there is no such flag for Redis port. And adding option to ray.init() may require more review.

Reply:
should this be prefixed? i'm not sure about the rule here. i can update this.